### PR TITLE
EDU-450 Support uploading video image poster

### PR DESCRIPTION
**Pull request by adrianaluchian**
Closes https://educandu.atlassian.net/browse/EDU-450

On a Prod dump, the report was:

```
Updated 17 lessons, 285 documents, 5052 documentRevisions
info [2022-03-28T18:03:28.041Z] [/src/stores/database.js] {
  event: 'migrated',
  name: 'educandu-2022-03-28-01-update-video-sections-content',
  durationSeconds: 18.555
}
```

### CODE REVIEW

File: src/plugins/video/info.js [hunk 1]
```diff
@@ -37,6 +41,6 @@ export default class Video {
   }
 
   getCdnResources(content) {
-    return content.type === SOURCE_TYPE.internal && content.url ? [content.url] : [];
+    return content.sourceType === SOURCE_TYPE.internal && content.sourceUrl ? [content.sourceUrl] : [];
```
Comment (ahelmberger) [inline src/plugins/video/info.js:44]: The poster image - if set - is also a CDN resource but is missing here.

File: src/plugins/video/video.yml [hunk 2]
```diff
@@ -37,3 +37,6 @@ width:
 copyrightInfos:
   en: Copyright infos
   de: Copyright Infos
+posterImageUrl:
+  en: Preview image URL
+  de: Vorschau Bild URL
```
Comment (ahelmberger) [inline src/plugins/video/video.yml:42]: We need hyphens between the words: "Vorschau-Bild-URL"

File: src/ui/mime-type-helper.js [hunk 3]
```diff
@@ -222,7 +222,7 @@ function getCategory(pathOrExtension) {
       return CATEGORY_AUDIO;
     case 'image':
       return CATEGORY_IMAGE;
-    case 'vido':
+    case 'video':
```
Comment (ahelmberger) [inline src/ui/mime-type-helper.js:225]: :scream:

File: src/plugins/video/video.yml (hunk 2 above)
Comment (adrianaluchian) [reply src/plugins/video/video.yml]: Ge-hyphenated, thank you

File: src/plugins/video/info.js (hunk 1 above)
Comment (adrianaluchian) [reply src/plugins/video/info.js]: Good catch, thanks! Updated

File: src/ui/mime-type-helper.js (hunk 3 above)
Comment (adrianaluchian) [reply src/ui/mime-type-helper.js:225]: Seems we always showed "Unbekannt" for videos 😅 Well, now we **know**

**Review by ahelmberger — APPROVED** [review]

**Pull request merged**